src/components/GroceryItem/GroceryItem.js

```diff
@@ -94,6 +105,19 @@ const GroceryItem = ({ item }) => {
           checked={isChecked}
         />
         {item.name} - Repurchase in {item.howSoon} days
+        <button
+          className="delete-button"
+          onClick={() => {
+            const confirmation = window.confirm(
+              'Did you mean to delete this item?',
+            );
+            if (confirmation) {
+              deleteItem(item.id);
```

Comment:
For readability, should we move this function up above the return. Then you can just change line 110 to: onClick={confirmation}